http_client.rst

```diff
@@ -154,6 +154,15 @@ brings most of the available options with type-hinted getters and setters::
             ->toArray()
     );
 
+.. tip::
+
+    Be aware that calling :method:`Symfony\\Component\\HttpClient\\HttpOptions::setHeaders` overrides all previous headers.
```

Comment:
I would update the code example above to call both the `setHeaders()` and `setHeader()` method and add inline comments explaining the difference.

Reply:
I've made a change in that direction. Tell me if there is anything I can improve further ;)